docs/modules/operation/pages/admin/rmi.adoc

```diff
@@ -25,10 +25,11 @@ This tells {page-component-title} to listen for RMI on port `18980`, and to list
 (Originally, RMI was used only for the legacy remote poller, so despite the property name mentioning the "opennms poller server" it applies to RMI as a whole.)
 Note that you must include the `-Djava.rmi.server.hostname=` option or {page-component-title} will accept connections on the RMI port but not be able to complete a valid connection.
```

Comment:
```suggestion
Note that you must include the `-Djava.rmi.server.hostname=` option, otherwise {page-component-title} will accept connections on the RMI port but not be able to complete a valid connection.
```